warehouse/clickhouse/clickhouse.go

```diff
@@ -200,6 +200,10 @@ func Init() {
 	pkgLogger = logger.NewLogger().Child("warehouse").Child("clickhouse")
 }
 
+func connectWithTunnel(cred CredentialsT, includeDBInConn bool) (*sql.DB, error) {
```

Comment:
Are we keeping this for future references?

Reply:
I'll remove this. We will add it when we handle it.